src/ape_geth/provider.py

```diff
@@ -399,7 +426,29 @@ def _log_connection(self, client_name: str):
         )
         logger.info(f"{msg} {suffix}.")
 
-    def _make_request(self, endpoint: str, parameters: List) -> Any:
+    def get_contract_creation_receipts(
+        self,
+        address: AddressType,
+        start_block: int = 0,
+        stop_block: Optional[int] = None,
+        contract_code: Optional[HexBytes] = None,
+    ) -> Iterator[ReceiptAPI]:
+        if self._ots_api_level is not False:
+            result = self._make_request("ots_getContractCreator", [address])
+            if not result:
```

Comment:
this can be tightened up, since ots returns null for non-contracts
```suggestion
            if result is None:
```

Reply:
done